server/src/database/ quieres/tasksQueries/getTasksSectionQuery.ts

```diff
@@ -0,0 +1,11 @@
+import connection from '../../config/connection';
+
+const getTasksSectionQuery = (id) => {
+  const sql = {
+    text: 'select * from tasks where sectionid = $1 ',
+    values: [id],
```

Comment:
```sectionId``` not  sectionid (camelCase)

Reply:
In the database written sectionid 🙄 so I write it

Reply:
rename it in database when you pass it as arguments ;)